### PR TITLE
feat: Enable JSII build tool

### PR DESCRIPTION
**Pull request by RomainMuller**
Configures building Java bindings using `npm run package`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,22 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "prepublish": "npm run build"
+    "build": "jsii",
+    "watch": "jsii -w",
+    "prepublish": "npm run build",
+    "build:bindings": "jsii-pacmak"
```
Comment (eladb) [inline package.json]: This should be called "package" per our conventions

Comment (RomainMuller) [reply package.json]: I might want to revisit those conventions (to be more node-idiomatic in certain ways). But regardless... Consistency is key 👌🏻

**Comment by RomainMuller** [comment]
Also - FYI I'm unable to merge here since you're the owner of that package (🤔). You'll have to DIY :)

**Comment by eladb** [comment]
Done already

**Pull request closed without merging**